docs/log-in-as-po.md

```diff
@@ -1,24 +1,55 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+Public officer can log in to TechPass using your non-SE GSIB device or your Government Managed Device (GMD).
+
+**To log in with a non-SE GSIB device**
+
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
 
-If you are signing in from your GMD, you will be prompted to authenticate your WOG account.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
 
-3. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
+2. Click **Log in with TechPass**.
 
-<kbd>![verify-wog-for-po](assets/images/access-sgts-services-using-techpass/verification-code-po.png)-->
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-tech.png)</kbd>
 
-You will now be prompted to approve your TechPass sign-in. A number will be shown on your browser.
+3. Sign in to your TechPass account or choose your TechPass account.
 
- <kbd>![number-mfa](assets/images/onboarding/po-non-se/mfa-number-displayed-on-screen.png)</kbd>
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+4. You will be prompted to authenticate your WOG account. A number will be shown on your browser.
```

Comment:
@kpslp there are three use cases:

you need to capture the exp of the user for all the three in this step.

Reply:
Use case 1:

1. Log in to GSIB.
2. try to access a resource behind TechPass.
3. If i am not logged into WOG account or session timed out, i will be prompted to enter pwd (GSIB pwd)
4. pwd success, then i need to authenticate by keying the number i see right?
5. no need to authenticate TP profile.
6. access the resource now.

Use case 2:

1. Log in to GSIB.
2. try to access a resource behind TechPass.
3. If i am already logged into WOG account (session still active)
4. Enter the authenticate  number  you see.
5. no need to authenticate TP profile.
6. access the resource now.


use case 3:

1. Log in to GMD.
2. try to access a resource behind TechPass.
3. enter WoG verification code (6 digit OTP)  and then enter the number  displayed for TP .
5. access the resource now.